### DISCUSSION TITLE
question about default volume when connect

### DISCUSSION THREAD

**Discussion by dongjin0521** [steam3d/MagicPods-Windows#128] [Q&A] [answered]
Hi, When I right click system tray icon and connect airpods,(not in windows setting)
default volume always setted to max.
Is there any solution?

**Comment by steam3d** [comment, marked as answer]
This is known windows issue.  You can also get 100% volume when you switch to the next music track. 
Read this https://www.reddit.com/r/airpods/comments/b6l5bm/airpods_2_windows_10_max_volume_fix/
It will fix the volume. Note after update windows can restore setting.

**Reply by dongjin0521** [reply]
wow! thanks a lot
